op-aws-sdk/key_manager.go

```diff
@@ -0,0 +1,81 @@
+package op_aws_sdk
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/urfave/cli"
+)
+
+const (
+	OP_NODE_P2P_SEQUENCER_KEY = "OP_NODE_P2P_SEQUENCER_KEY"
+	OP_BATCHER_SIGN_KEY       = "OP_BATCHER_SIGN_KEY"
+	OP_PROPOSER_SIGN_KEY      = "OP_PROPOSER_SIGN_KEY"
+	AWS_KEY_JSON_NAME         = "pk"
+)
+
+func KeyManager(context context.Context, ctx *cli.Context, keyType string) error {
+	secretName := ""
+	awsRegion := ""
+	flagName := ""
+	log.Info("Key manager ", "keyType", keyType)
+	switch keyType {
+	case OP_NODE_P2P_SEQUENCER_KEY:
+		secretName = "OP_NODE_AWS_P2P_SECRET_NAME"
+		awsRegion = "OP_NODE_AWS_P2P_SECRET_REGION"
+		flagName = "p2p.sequencer.key"
+	case OP_BATCHER_SIGN_KEY:
+		secretName = "OP_BATCHER_AWS_SECRET_NAME"
+		awsRegion = "OP_BATCHER_AWS_SECRET_REGION"
+		flagName = "private-key"
+	case OP_PROPOSER_SIGN_KEY:
+		secretName = "OP_PROPOSER_AWS_SECRET_NAME"
+		awsRegion = "OP_PROPOSER_AWS_SECRET_REGION"
+		flagName = "private-key"
+	default:
+		log.Error("Key manager ", "error keyType", keyType)
+		return errors.New("unknown key type")
+	}
+	return load(context, ctx, awsRegion, secretName, flagName)
+}
+func load(context context.Context, ctx *cli.Context, awsRegion string, secretName string, flagName string) error {
+	name := os.Getenv(secretName)
+	region := os.Getenv(awsRegion)
+	if name != "" {
+		config, err := config.LoadDefaultConfig(context, config.WithRegion(region))
```

Comment:
The variable name `config` is shadowing the global package name `config`, which goes against best practices. It is recommended to rename the variable.